### PR TITLE
Suggest adding a Radar link to internal contributors when running git-webkit pr

### PR DESCRIPTION
**Pull request by alancoon**
#### b644cfdda4fc18287487baaded4e288b3bc80364
<pre>
Suggest adding a Radar link to internal contributors when running git-webkit pr
<a href="https://bugs.webkit.org/show_bug.cgi?id=242914">https://bugs.webkit.org/show_bug.cgi?id=242914</a>
rdar://97276993

Reviewed by Jonathan Bedard.

* Tools/Scripts/hooks/prepare-commit-msg: add logic to suggest internal contributors
add a Radar link to their commits
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/squash.py:
(Squash.squash_commit): drive-by typo fix

Canonical link: <a href="https://commits.webkit.org/252672@main">https://commits.webkit.org/252672@main</a>
</pre>

### CODE REVIEW

**Comment by alancoon** [comment]
This PR is to add logic to the  `prepare-commit-msg` Git hook that will alter the template to look like the following, if it discovers that the user has radarclient installed, and is thus an internal Apple Employee:
```
    Need a short description (OOPS!).
    Need the bug URL (OOPS!).
    Heavily recommended: include a Radar link with no title (OOPS!).

    Reviewed by NOBODY (OOPS!).

    ...
```

Since `prepare-commit-msg` is copied over while running `git-webkit setup`, this will not affect people's current checkouts, only new ones, or people who re-run `git-webkit setup`. In the future I plan on implementing logic to detect that someone's `prepare-commits-msg` hook is outdated and copying it over while running `git-webkit pr`.

File: Tools/Scripts/hooks/prepare-commit-msg [hunk 1]
```diff
@@ -10,6 +10,29 @@ SCRIPTS = os.path.dirname(os.path.dirname(LOCATION))
...
 sys.path.append(SCRIPTS)
 from webkitpy.common.checkout.diff_parser import DiffParser
+from webkitbugspy import radar
+
+
+def get_bugs_string():
+    """Determines what bug URLs to fill or suggest in a WebKit commit message."""
+    env = os.environ
+    need_the_bug_url = 'Need the bug URL (OOPS!).'
+    need_the_radar = 'Heavily recommended: include a Radar link with no title (OOPS!).'
```
Comment (JonWBedard) [inline Tools/Scripts/hooks/prepare-commit-msg]: Would just change the to `Include a Radar link (OOPS!)`

**Review by JonWBedard — APPROVED** [review]

File: Tools/Scripts/hooks/prepare-commit-msg (hunk 1 above)
Comment (youssefsoli) [reply Tools/Scripts/hooks/prepare-commit-msg]: This would be a bit awkward to deal with until automatic radar importing is enabled w.r.t bugzilla right?

Comment (JonWBedard) [reply Tools/Scripts/hooks/prepare-commit-msg]: We already have some automatic radar importing, in that if you give `git-webkit` a bugzilla which is cloned into radar, we'll pick up the radar too. @alancoon and I do need to make this particular behavior cover a few more cases.

**Pull request merged**